scripts/compare_lb_data_file.py

```diff
@@ -0,0 +1,36 @@
+import argparse
+import json
+import sys
+from deepdiff import DeepDiff
+
+def compare(file_to_validate, reference_file):
+    """
+    Compares file to validate wih reference
+    """
+    with open(file_to_validate) as val_file, open(reference_file) as ref_file:
+        to_validate = json.load(val_file)
+        reference = json.load(ref_file)
+        diff = DeepDiff(to_validate, reference, report_repetition=True, math_epsilon=0.1)
+        is_valid = not len(diff.affected_paths)
+
+        if not is_valid:
```

Comment:
I think checking if `diff` is not empty should be enough:
```suggestion
        if diff:
```

Reply:
Fixed.